howto/application/test.md

```diff
@@ -93,7 +93,10 @@ In this example we use the following application `manifest.yaml`:
 
 ```
 name: app
-instance-type: a4.3
+resources:
+  cpus: 4
+  memory: 4GB
+  disk-size: 8GB
```

Comment:
Is this intended? Shouldn't this be 3GB of memory and disk-size?

Reply:
Ah sorry, that was for a different example where more resources where required. Fixed it.